packages/ubuntu_provision/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   args: ^2.4.2
   async: ^2.11.0
   collection: ^1.17.0
+  dartx: ^1.2.0
```

Comment:
Is it really worth adding a dependency for only `removePrefix`? :sweat_smile: 
Something like `replaceFirst(r'^\/', '')` can be used instead of `removePrefix('/');`, or simply `replaceFirst('/', '');` since all will start with `/`.

Reply:
Hm, we got it in ubuntu_bootstrap already and it's listed in the melos.yaml as well, so I guess it doesn't hurt? But I'm happy to adopt your solution if that's cleaner

Reply:
It doesn't seem to be used in ubuntu_bootstrap anymore, it's just in the pubspec, so probably better to just remove it everywhere?

Reply:
It's used in various places, but in all cases there are replacements from `dart:core` or `collection`

Reply:
Unexpected subtle differences between `elementAtOrNull` from `collection` vs `dartx` :/
It's probably a good idea to remove dartx :+1: